### PR TITLE
added a few data related libs

### PR DESCRIPTION
**Pull request by Moshe-Immerman**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -703,6 +706,7 @@ A curated list of awesome Java frameworks, libraries and software.
 * [GraphStream](http://graphstream-project.org/) - Library for modeling and analysis of dynamic graphs.
 * [JGraphT](https://github.com/jgrapht/jgrapht) - Graph library that provides mathematical graph-theory objects and algorithms.
 * [JGraphX](https://github.com/jgraph/jgraphx) - Library for visualisation (mainly Swing) and interaction with node-edge graphs.
+* [Tablesaw](https://github.com/lwhite1/tablesaw) - The simplest way to slice data in Java.
```
Comment (akullpp) [inline README.md]: Description doesn't really tell anything.

File: README.md [hunk 2]
```diff
@@ -267,9 +268,11 @@ A curated list of awesome Java frameworks, libraries and software.
 * [Apache Orc](https://orc.apache.org/) - Fast and efficient columnar storage format for hadoop based workloads.
 * [Apache Parquet](http://parquet.apache.org/) - Columnar storage format based on assembly algorithms from the Dremel paper by Google.
 * [Apache Thrift](https://thrift.apache.org/) - Data interchange format that originated at Facebook.
+* [Big Queue](https://github.com/bulldog2011/bigqueue) - A big, fast and persistent queue based on memory mapped file.
 * [Persistent Collection](http://pcollections.org/) - Persistent and immutable analogue of the Java Collections Framework.
 * [Protobuf](https://github.com/google/protobuf) - Google's data interchange format.
 * [SBE](https://github.com/real-logic/simple-binary-encoding) - Simple Binary Encoding, one of the fastest message formats around.
+* [Tape](https://github.com/square/tape) - A lightning fast, transactional, file-based FIFO for Android and Java.
```
Comment (akullpp) [inline README.md]: Remove "for Android and Java"

File: README.md [hunk 3]
```diff
@@ -255,6 +255,7 @@ A curated list of awesome Java frameworks, libraries and software.
 * [Querydsl](http://www.querydsl.com/) - Typesafe unified queries.
 * [Realm](https://github.com/realm/realm-java) - Mobile database to run directly inside phones, tablets or wearables.
 * [Redisson](https://github.com/mrniko/redisson) - Allows for distributed and scalable data structures on top of a Redis server.
+* [requery](https://github.com/requery/requery) - modern SQL based query & persistence for Java / Kotlin / Android.
```
Comment (akullpp) [inline README.md]: Remove "Java / Kotlin / Android"

Comment (Moshe-Immerman) [inline README.md]: @akullpp done

**Pull request merged**